app/com/linkedin/drelephant/ElephantRunner.java

```diff
@@ -188,10 +310,160 @@ private void waitInterval() {
     }
   }
 
+  private void stopFurtherAnalysis(String appId) {
```

Comment:
stopRealtimeAnalysis seems to be a better name as it is used only for realTime analysis.